### PR TITLE
Update changelog (content+format). Update MSRV text.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This brings in some of the updates to the format that we're using in other projects as well as mentioning the MSRV.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,27 +1,53 @@
...
+- An edge case in `mindist` was fixed. ([#334] by [@platlas])
+- Allow lines in simplify input. ([#343] by [@raphlinus])
+- Documentation for `Arc.perimeter` was corrected. ([#354] by [@simoncozens])
 
-## 0.11.0 (2024-02-14)
+## [0.11.0][] (2024-02-14)
+
+This release has an [MSRV][] of 1.65.
 
 - ...
```
Comment (DJMcNab) [inline CHANGELOG.md:36]:
```suggestion
Note: A changelog was not kept for or before this release
```
or something similar?

Just an ellipsis isn't very helpful here,

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,27 +1,53 @@
...
+This changelog follows the patterns described here: <https://keepachangelog.com/en/>.
 
 Subheadings to categorize changes are `added, changed, deprecated, removed, fixed, security`.
 
 -->
 
+# Changelog
+
+The latest published Kurbo release is [0.11.0](#0110) which was released on 2024-02-14.
+You can find its changes [documented below](#0110).
```
Comment (DJMcNab) [inline CHANGELOG.md:12]: These links don't work for me

**Review by DJMcNab — APPROVED** [review]
Some small tweaks

File: CHANGELOG.md (hunk 2 above)
Comment (waywardmonkeys) [reply CHANGELOG.md]: Whoops, fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (waywardmonkeys) [reply CHANGELOG.md]: Done.

**Pull request merged**